README.md

```diff
@@ -818,16 +819,25 @@ Refer to the relevant Yaskawa Motoman documentation for more information on how
 
 After robot-calibration, the transform between the shared `world` frame and each robot's `base` frame will be known, and MotoROS2 will include it in the transforms it broadcasts.
 
-### Some group combinations won't publish data
+### Best-effort reliability topics fail to publish large messages
 
-**Description**: it has been observed that an R1+R2+S1 system will not publish data `/joint_states` or `/tf`.
-All other topics and services work as expected.
+**Description**: Topics with QoS set to best-effort reliability will fail to publish large messages.
+This behavior is most commonly observed with `/joint_states`.
+As an example, if your system has a large number of joints or long joint names which result in a large message, `/joint_states` will not be published.
+This can occur with other best-effort reliability topics as well.
+The root cause of this behavior is described [here](https://github.com/eProsima/Micro-XRCE-DDS-Client/issues/394#issuecomment-2519526357).
 
-**Work-around**: such a multi-group system would need to be broken up into a independent systems.
-(E.g. `R1+R2+S1` would be broken up into `R1+S1` and another `R1`).
+**Work-arounds**: There are two options for fixing this behavior.
 
-The cause of this behavior is unknown.
-The issue is being investigated.
+1. Reduce the length of your messages.
+For example, you can reduce the length of the `/joint_states` message by modifying `joint_names` in [the configuration file](#configuration) to reduce the length of joint names.
```

Comment:
The message is `sensor_msgs/JointState`, the topic is `joint_states`.

If referring to configuration items in the configuration file, I'd suggest using a perma-link to the exact item, to avoid users having to search for them.